submissions/api.py

```diff
@@ -626,7 +626,7 @@ def get_score(student_item):
 
 
 def get_scores(course_id, student_id):
-    """Return a dict mapping item_ids -> (points_earned, points_possible).
+    """Return a dict mapping item_ids -> [ScoreSerializer(ScoreSummary)].
```

Comment:
Since we're taking the `data` property in particular of a Serializer, could this more accurately be `Return a dict mapping item_ids -> JSON-style object representation`?